beacon-chain/state/v1/error.go

```diff
@@ -0,0 +1,10 @@
+package v1
```

Comment:
Perhaps these can be state-level errors instead of each vi repeating them

Reply:
I think it's OK to duplicate them. They will be gone eventually when the native state takes over.